lib/cartopy/examples/utm_all_zones.py

```diff
@@ -0,0 +1,47 @@
+"""
+Displaying all 60 zones of the UTM projection
+---------------------------------------------
+
+This example displays all 60 zones of the Universal Transverse Mercator
+projection next to each other in a figure.
+
+First we create a figure with 60 subplots in one row.
+Next we set the projection of each axis in the figure to a specific UTM zone.
+Then we add coastlines, gridlines and the number of the zone.
+Finally we add a supertitle and display the figure.
+"""
+__tags__ = ['Lines and polygons']
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+
+def main():
+    # Create a list of integers from 1 - 60
+    zones = range(1, 61)
```

Comment:
This is not a list on Python 3; I'd just inline it into the `for`-loop.

Reply:
1. In Python 3 `range()` is not a list but it will work in the `for` loop just the same, right?

2. I separated the list of zones from the `for` loop to make it easy for someone to try out their own list of zones.
e.g. List of anomalous zones from the [comment](https://github.com/SciTools/cartopy/pull/954#discussion_r152208271) above